hack/Dockerfile_proto

```diff
@@ -0,0 +1,43 @@
+# Install protoc
```

Comment:
let's move this under a dir in hack/proto/Dockerfile so that syntax highlighting can work

Reply:
done